backend/Application/Api/GraphQL/Import/PaydayImportHandler.cs

```diff
@@ -45,8 +45,15 @@ public async Task<BlockImportPaydayStatus> UpdatePaydayStatus(BlockDataPayload p
             }
             else if (status.NextPaydayTime != rewardStatus.NextPaydayTime)
             {
-                var duration = status.NextPaydayTime - status.PaydayStartTime;
-                var result = new FirstBlockAfterPayday(status.NextPaydayTime, Convert.ToInt64(duration.TotalSeconds));
+                var duration = Convert.ToInt64((status.NextPaydayTime - status.PaydayStartTime).TotalSeconds);
+
+                // Upon update to protocol 5 the time was shifted by ~5 secs on Testnet
```

Comment:
This is not correct. How come you arrived at this understanding?

Reply:
There was a shift of 287 seconds

Reply:
The comment says 5 secs.

Reply:
fixed